app/components/background.tsx

```diff
@@ -1,10 +1,11 @@
 /**
- * Fixed background component
```

Comment:
Setting the scrolling background to only scroll if the user is motion-safe, ie they do not have prefer-reduced-motion set on their device.